client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/notices/notices.tsx

```diff
@@ -0,0 +1,38 @@
+import { SnackbarList, withNotices } from '@wordpress/components';
+import i18n from 'i18n-calypso';
+import { ReactNode } from 'react';
+import type { Pattern } from '../types';
+import './notices.scss';
+
+type NoticesProps = {
+	noticeList: withNotices.Props[ 'noticeList' ];
+	noticeOperations: withNotices.Props[ 'noticeOperations' ];
+	noticeUI?: ReactNode;
+};
+
+const Notices = ( { noticeList, noticeOperations }: NoticesProps ) => {
+	const onRemoveNotice = ( id: string ) => {
+		noticeOperations.removeNotice( id );
+	};
+
+	return <SnackbarList notices={ noticeList } onRemove={ onRemoveNotice } />;
+};
+
+export const getNoticeContent = ( action: string, pattern: Pattern ) => {
+	const actions: { [ key: string ]: any } = {
+		add: i18n.translate( 'Pattern "%(categoryName)s" added.', {
+			comment: 'A pattern is added to the list using the pattern category as name',
+			args: { categoryName: pattern.category?.label },
+		} ),
+		replace: i18n.translate( 'Pattern "%(categoryName)s" replaced.', {
+			args: { categoryName: pattern.category?.label },
+		} ),
+		remove: i18n.translate( 'Pattern "%(categoryName)s" removed.', {
```

Comment:
We are not showing a notice when patterns are removed, but I see value in that. Should we add it? I'll remove this line otherwise.

Reply:
I added a commit to include the notice when section patterns are removed.